.github/workflows/smoking_test.yaml

```diff
@@ -0,0 +1,58 @@
+name: Smoking test suit
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'pkg/*'
+env:
+  RUSTFLAGS: "-D warnings"
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  TESTNET_SNAPSHOT_URL: ${{secrets.TESTNET_DATA_SNAPSHOT}}
+  CARGO_TARGET_DIR: ${{ github.workspace }}/../target
+
+jobs:
+   Start_and_sync:
+    runs-on: [self-hosted,Linux]
+    steps:
```

Comment:
I think we can use `workflow_dispatch` and `repository_dispatch` to trigger this workflow with the input to the CKB package URL. In `package.yml` we can trigger smoking test with the Linux package URL. So we can save the time to build CKB and just use the provided package to run the smoking test.